lib/jnpr/jsnapy/snap.py

```diff
@@ -214,10 +216,12 @@ def run_rpc(self, test_file, t, formats, dev, output_file, hostname, db):
         self.logger_snap.debug(colorama.Fore.BLUE +
                                "Tests Included : %s " %t,
                                extra=self.log_detail)
-        self.logger_snap.info(colorama.Fore.BLUE +
-                              "Taking snapshot of RPC: %s" %
-                              rpc,
-                              extra=self.log_detail)
+        if rpc not in rpc_list:
```

Comment:
Here also the same problem as above is there. In the `else` scenario the snapshot is still taken. Stop the snapshot procedure in the else scenario.